src/ciflow-bot.ts

```diff
@@ -0,0 +1,220 @@
+import * as probot from 'probot';
+
```

Comment:
A top level comment describing the general strategy for the ci flow bot would be useful for later code readers